### ISSUE TITLE
can i install at heroku ?

### ISSUE THREAD

**Issue by bhoeyong** [rafaroman18/TorrentRaspBot#1]
hi bro..this script i can install at heroku ?

**Comment by rafaroman18** [comment]
Of course! The problem is that is not finished yet! 😅 My idea is to have it finished by the end of the month (im very busy with college)

**Comment by bhoeyong** [comment]
Heroku now hard to install...After u find teach me how to use it.... Thanks

**Comment by rafaroman18** [comment]
Yeah! Btw, i would modify the `README.md` with a tutorial of how to do this bot for personal use 👍

**Comment by bhoeyong** [comment]
hi bro...can use it already ?

**Comment by rafaroman18** [comment]
Nope. Until more or less November I cant avance on it. Dont worry, I will tell you when it's finished!

**Comment by a092devs** [comment]
any update yet?

**Comment by rafaroman18** [comment]
Sorry...The college is killing me and I have no much time... 😅

I have achieved that the bot can receive the link, and download the file in a proper way such that the program waits until the file is downloaded. After this, it checks the 'file' type and if it is a torrent file, then send a message to the user. 

From here, the things to finish are the transmission (linux program to download torrents) commands (which I already know them but I haven't included yet) and the upload to the Google Drive Folder (this step is very simple because I only have to move the file into a simple folder of the system, because I use the rclone method).

One more thing to comment is that when it is finished, I will update the Readme.md such that I explain EVERYTHING: how to use, setup and a little memory about the making process.

P.S. @bhoeyong I didn't forget about you, I just want to try to explain you how to install when the whole Bot is finished 👍

**Comment by a092devs** [comment]
Take your time, just wanted to be tagged in the progress!
Thanks
